tests/unit/OnyxDerivedTest.tsx

```diff
@@ -12,6 +16,11 @@ import {createRandomReport} from '../utils/collections/reports';
 import createRandomTransaction from '../utils/collections/transaction';
 import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
 
+const renderLocaleContextProvider = () => {
+    // @ts-expect-error TS2741: Property 'children' is missing
+    render(<ComposeProviders components={[OnyxListItemProvider, LocaleContextProvider]} />);
+};
+
```

Comment:
NAB: The `@ts-expect-error` comment suggests this is a hacky solution, tests should have proper setup:
```suggestion
function renderWithLocaleProvider(component: React.ReactElement) {
    return render(
        <ComposeProviders components={[OnyxListItemProvider, LocaleContextProvider]}>
            {component}
        </ComposeProviders>
    );
}
```

Example usage for the `getLastActorDisplayName() - should return correct display name` test:

```ts
renderWithLocaleProvider(
    <OptionsListContextProvider>
        <ScreenWrapper testID="test">TEST</ScreenWrapper>
    </OptionsListContextProvider>
);
// ... rest of test
```